dev-libs/libpeas/libpeas-2.0.3.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-1 )
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit gnome.org lua-single meson python-single-r1 vala virtualx xdg
+
+DESCRIPTION="A GObject plugins library"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/libpeas"
+
+LICENSE="LGPL-2.1+"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="gjs glade +gtk gtk-doc lua +python vala"
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	>=dev-libs/glib-2.44:2
+	>=dev-libs/gobject-introspection-1.39:=
+	gjs? (
+		>=dev-libs/gjs-1.77.1
+		>=dev-lang/spidermonkey-115
```

Comment:
This should be dev-lang/spidermonkey:115 slot

And probably gjs be raised to 1.77.2 as suggested in https://github.com/gentoo/gentoo/pull/32202#discussion_r1314967052

Reply:
Done in https://github.com/gentoo/gentoo/pull/34835/commits/6cddcffa77b0823b7a569da486e87228203288b7